libs/model/src/community/GetCommunity.query.ts

```diff
@@ -10,20 +11,94 @@ export function GetCommunity(): Query<typeof schemas.GetCommunity> {
     secure: false,
     body: async ({ payload }) => {
       const where = { id: payload.id };
-      const include: Includeable[] = [];
+      const include: Includeable[] = [
+        {
+          model: models.CommunityBanner,
+        },
+        {
+          model: models.CommunityStake,
+        },
+        {
+          model: models.CommunityTags,
+          include: [
+            {
+              model: models.Tags,
+            },
+          ],
+        },
+      ];
       if (payload.include_node_info) {
         include.push({
           model: models.ChainNode,
           required: true,
         });
       }
 
-      return (
-        await models.Community.findOne({
-          where,
-          include,
-        })
-      )?.toJSON();
+      const result = await models.Community.findOne({
+        where,
+        include,
+      });
+
+      if (!result) {
+        return;
+      }
+
+      const [
+        adminsAndMods,
+        numVotingThreads,
+        numTotalThreads,
+        communityBanner,
+      ] = await (<
+        Promise<
+          [
+            Array<{ address: string; role: 'admin' | 'moderator' }>,
+            number,
+            number,
+            { banner_text: string } | undefined,
+          ]
+        >
+      >Promise.all([
+        models.Address.findAll({
+          where: {
+            community_id: payload.id,
+            [Op.or]: [{ role: 'admin' }, { role: 'moderator' }],
+          },
+          attributes: ['address', 'role'],
+        }),
+        models.Thread.count({
+          where: {
+            community_id: payload.id,
+            stage: 'voting',
+          },
+        }),
+        models.Thread.count({
+          where: {
+            community_id: payload.id,
+            marked_as_spam_at: null,
+          },
+        }),
+        models.CommunityBanner.findOne({
+          where: {
+            community_id: payload.id,
+          },
+        }),
+      ]));
+
+      return {
+        ...result.toJSON(),
+        adminsAndMods,
+        numVotingThreads,
+        numTotalThreads,
+        communityBanner: communityBanner?.banner_text,
+      } as CommunityAttributes & {
+        numVotingThreads: number;
+        numTotalThreads: number;
+        adminsAndMods: Array<{
```

Comment:
NIT: We don't use Array type too often. We usually use the shorthand syntax []. So like:
```typescript
adminsAndMods:{
  address: string;
  role: 'admin' | 'moderator';
}[]
```

Reply:
I'm not sure I agree that this is the best way to render the type. It feels easy to miss.